docs/theme-specification.md

```diff
@@ -7,49 +7,88 @@ The object itself is not coupled to Styled System's implementation and can be us
 ## Scale Objects
 
 Many CSS style properties accept open-ended values like lengths, colors, and font names.
-In order to create a consistent styling system, the theme object is centered around the idea of scales, such as a typographic (font-size) scale, a spacing scale for margin and padding, and a color object.
-These scales can be defined in multiple ways depending on needs, but tend to use arrays for ordinal values like font sizes, or plain objects for named values like colors, with the option of nesting objects for more complex systems.
+In order to create a consistent styling system, the theme object is centered around the idea of scales, such as a typographic (font-size) scale, a spacing scale for margin and padding, and a color object. Scales must be JSON serializable - this way can be used in JS and CSS but possibly even cross platform.
 
-```js
-// example fontSizes scale as an array
-fontSizes: [
-  12, 14, 16, 20, 24, 32
-]
-```
+These scales can be defined after their correspective CSS property names when possible and use arrays for variants or ordinal values like font sizes. Short hand values are expressed as nested objects. Special categories can be defined to describe multiple CSS properties whichmust be listed in the `aliases` part of the object.
+
+Each scale must contain exactly 7 variants, where the first one is the base value.
+(not sure what is the best way to define base + variants values though - maybe your idea of singular and plural is much better).
+
+The idea is to mirror the HTML heading scale h1-6 + a base value - 6 variants are normally good enough. I might be wrong though, is 10 or Pantone-style better?
```

Comment:
specifically for color maybe the human readable keys should be the standard color keywords https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords - the color doesn't have to match the standard hex value and be slightly different but the keyword (property name) would at least be ±universal